controllers/buildpipeline/buildpipeline_adapter.go

```diff
@@ -32,9 +32,13 @@ import (
 	"github.com/redhat-appstudio/operator-toolkit/controller"
 	"github.com/redhat-appstudio/operator-toolkit/metadata"
 	tektonv1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"k8s.io/client-go/util/retry"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
+const finalizer string = "build.appstudio.openshift.io/pipelinerun"
```

Comment:
Can we define this in the same way that [IntegrationPipelineRunFinalizer](https://github.com/redhat-appstudio/integration-service/blob/96dc306bf01e6596e2518c2eec29dcccc0201b47/helpers/finalizers.go#L20) is defined?